bundle/keys_test.go

```diff
@@ -182,6 +212,18 @@ EXrJfkELSzO66/ZSjyyWEczXHLyr+Q719BsaGsxie117zSNF6B6UXiitjCr/qQ==
 			t.Fatalf("Expected key type *rsa.PrivateKey but got %T", result)
 		}
 
+		sc = NewSigningConfig(filepath.Join(rootDir, "pkcs8.pem"), "", "")
```

Comment:
It's not this PR's focus at all, but wouldn't this test benefit from a few `t.Run("description", func (t *testing.T) { ... })` blocks?

Reply:
Yeah that could be - let's restructure these if/when we extend the functionality of the `keys` object.